src/components/vmd-commune-or-departement-selector.component.ts

```diff
@@ -134,17 +134,23 @@ export class VmdCommuneOrDepartmentSelectorComponent extends LitElement {
 
     render() {
         return html`
-          <form class="autocomplete ${classMap({'_open': this.showDropdown, '_withButton': this.filter !== ''})}"
+          <form class="row align-items-center autocomplete ${classMap({'_open': this.showDropdown, '_withButton': this.filter !== ''})}"
                 @submit="${this.handleSubmit}">
-            <input type="search" class="autocomplete-input"
-                   required
-                   @focusin="${() => { this.inputHasFocus = true; window.scroll({ top: this.offsetTop - 32, behavior: 'smooth' }); }}"
-                   @focusout="${this.hideDropdownWhenInputHasNotFocus}"
-                   @keydown="${this.handleKeydown}"
-                   @keyup="${this.valueChanged}"
-                   .value="${this.filter}"
-                   placeholder="Commune, Code postal, Département..."
-            />
+            <label for="searchAppointment-searchbar" class="col-sm-24 col-md-auto mb-md-1 label-for-search">
+                Localisation :
+            </label>
+            <div class="col">
```

Comment:
Hello,
ya un padding en trop à gauche du `input` sur mobile
![image](https://user-images.githubusercontent.com/1426314/117576685-14ede100-b0e7-11eb-85eb-24c15dddcba3.png)
```suggestion
            <label for="searchAppointment-searchbar" class="col-sm-24 col-md-auto mb-md-1 ps-0 pe-1 label-for-search">
                Localisation :
            </label>
            <div class="col px-0">
```

Reply:
C'est plus le Label qui n'a pas de padding left. 
Le input est pour le coup bien centré.

Reply:
Merci pour tes retours :)